CONTRIBUTING.md

```diff
@@ -71,18 +88,33 @@ Remember to always work in a branch of your local copy, as you might otherwise h
 
 Please also see [GitHub workflow](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#github-workflow) section of general project contributing guide.
 
-### Running the tests
+## Development
+
+### Tools used
+
+- [NPM](https://npmjs.com)
+- [TypeScript](https://www.typescriptlang.org/)
+- [lerna](https://github.com/lerna/lerna) to manage dependencies, compilations, and links between packages. Most lerna commands should be run by calling the provided npm scripts.
```

Comment:
I think this is out of date now. AFAIK `lerna` is only now used for (a) `lerna run ...` because that supports parallel execution of a command in many workspace package dirs and (b) `lerna publish ...` for the release process (though I'm not sure on this one).

Perhaps other devs use other `lerna ...` commands for their work?

Reply:
I'll be honest, I'm not positive exactly how lerna and npm workspaces work together in this repo. I noticed lerna is still in commands in the `package.json` which is what led me to believe it's still in use for various things... but I do know that running lerna commands directly (instead of the npm command that calls lerna) causes problems and doesn't work as expected so I'm very much open to someone telling me what I have wrong here! 😅

Reply:
Yah, I had hoped to drop lerna entirely when starting using npm workspaces, but `npm run ...` does not support the parallel running that `lerna run ...` does.  Also `lerna ...` was being used -- at least I think so -- in some of the release process that I don't know well at all.